src/server/actions/study-job.actions.ts

```diff
@@ -73,21 +71,30 @@ export const rejectStudyJobResultsAction = memberAction(async (info) => {
 
 export const loadStudyJobAction = userAction(async (studyJobIdentifier) => {
     const userId = await getUserIdFromActionContext()
+
     const jobInfo = await db
         .selectFrom('studyJob')
         .innerJoin('study', 'study.id', 'studyJob.studyId')
         .innerJoin('member', 'study.memberId', 'member.id')
         .innerJoin('memberUser', (join) =>
             join.on('memberUser.userId', '=', userId).onRef('memberUser.memberId', '=', 'study.memberId'),
         )
+        .leftJoin('jobStatusChange', (join) =>
+            join
+                .on('jobStatusChange.studyJobId', '=', 'studyJob.id')
+                .on('jobStatusChange.status', 'in', ['RESULTS-APPROVED', 'RESULTS-REJECTED']),
+        )
         .select([
             'studyJob.id as studyJobId',
             'studyJob.studyId',
             'studyJob.createdAt',
             'study.title as studyTitle',
             'member.slug as memberSlug',
+            'jobStatusChange.status as jobStatus',
+            'jobStatusChange.createdAt as jobStatusCreatedAt',
         ])
         .where('studyJob.id', '=', studyJobIdentifier)
+        .orderBy('jobStatusChange.createdAt', 'desc')
```

Comment:
I don't think you need the orderBy here.  there should only be a single `jobStatusChange` row with `APPROVED`/`REJECTED`, in fact we have more than one then the query will have duplicate rows and we'd need to use a sub select vs just a join